mainnet-dryrun-2/gentx/vinjan.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"vinjan","identity":"7C66E36EA2B71F68","website":"https://service.vinjan.xyz","security_contact":"","details":"ðŸŽ‰Proffesional Stake \u0026 Node ValidatorðŸŽ‰"},"commission":{"rate":"0.010000000000000000","max_rate":"0.100000000000000000","max_change_rate":"0.100000000000000000"},"min_self_delegation":"1","delegator_address":"archway1lg68f0x669kd4ga50mtc320lkp53ddkzz9ufpe","validator_address":"archwayvaloper1lg68f0x669kd4ga50mtc320lkp53ddkzzp4tvs","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"Vpma+TKypg8M0zjngPXfHJlyDr1y9cmHQ9DSXRFshb4="},"value":{"denom":"aarch","amount":"9500000000000000000"}}],"memo":"61bdbe852a4b52ce99fd85e7389c2641c3c18671@38.242.153.36:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A1JHq7t5iohpU+J7DH80XCzPj1q0ull/rYsdIHn3D33W"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[{"denom":"aarch","amount":"180000000000000000"}],"gas_limit":"200000","payer":"","granter":""}},"signatures":["g6lcnQtfN8s7qRDGHD4v8nEGur+GqBDEH6pouPym47xE9XQ6tUrdKwh0p/18V/DrhRzHRrZWfE153BqB6neYbw=="]}
```

Comment:
Validation fails with
```
Error: failed to get genesis app state from config: account archway1lg68f0x669kd4ga50mtc320lkp53ddkzz9ufpe balance not in genesis state
```
The address wasn't collected for mainnet-dry-run-2, PR can be closed